Dev-Tools/aws/tool.md

```diff
@@ -4,16 +4,70 @@
 
 [Home](../../Readme.md) / [Dev Tools](../dev-tools.md) / [AWS](tool.md)
 
-Add the description here. It should be mostly professional and be on a low enough language that someone like Brian can get a brief understanding of what it is and why we use it. For an example of how these look, go to [teams](../microsoft-teams/tool.md) or [basecamp](../basecamp/tool.md). I recommend that all pictures go in as inline html `<img src="./images/logo.png" width=100px alt="logo"/>`. Widths should max at 600px and heights at 500px.
+AWS is our cloud service. It is separate from all IT infrastructure, and works as our main system for anything that is hosted. If you are wondering where any data or applications live, they live in AWS.
+
+It handles our frontend (websites), backend (business logic) and database (data storage).
 
 ## Installation
 
-If you need to install anything add the steps in here. If is just a website, add the link to the description, and remove this section.
+There is no installation needed for AWS. AWS is a web service. Depending on your exec, they will either issue everyone on the team an account, or only give out accounts to specific people. Please contact your exec to understand if you have an account.
+
+https://us-east-2.signin.aws.amazon.com/
+
+Our account ID: 177987852765
 
 ## How to use
 
 This will be the most important section. Describe (with pictures) how to set up projects, navigate the web app, and troubleshoot. Anything that can be useful should exist here.
```

Comment:
Can you remove this line? It was meant as description of what to put in this section.

Reply:
Let me specify line 21

Reply:
My b